src/wizards/clusters.ts

```diff
@@ -419,15 +420,7 @@ function saveCluster(update: boolean, data: any, cluster: Cluster, clusterSettin
 function saveClusters(update: boolean, clusters: Cluster[], clusterSettings: ClusterSettings, clientAccessor: ClientAccessor, explorer: KafkaExplorer) {
     try {
         // Save collected clusters in settings.
-        let createdClusterNames = '';
-        for (const cluster of clusters) {
-            clusterSettings.upsert(cluster);
-            clientAccessor.remove(cluster.id);
-            if (createdClusterNames !== '') {
-                createdClusterNames += '\', \'';
-            }
-            createdClusterNames += cluster.name;
-        }
+        let createdClusterNames = getNames(clusters);

```

Comment:
I think wizard création should wait for SaveClusterCommandHandler command and after we could consume it to savez clusters

Reply:
yeah I had something like that before the new wizard, but after rebasing, I just wanted to push something that compiles